package.json

```diff
@@ -1,44 +1,42 @@
 {
     "name": "gpii-handlebars",
-    "version": "1.1.5",
+    "version": "2.1.0-dev.20190923T144705Z.38550ff.GPII-4100",
```

Comment:
Looks like you've accidentally managed to commit a dev release version into package.json

Reply:
Thanks, fixed.